src/cluster.ts

```diff
@@ -80,6 +90,103 @@ export type GetClusterMetadataCallback = (
   apiResponse?: IOperation | null
 ) => void;
 
+export type BackupTimestamp = google.protobuf.ITimestamp | PreciseDate | Date;
+export interface ModifiableBackupFields {
+  /**
+   * The ITimestamp (Date or PreciseDate will be converted) representing
+   * when the backup will automatically be deleted. This must be at a
+   * minimum 6 hours from the time of the backup request and a maximum of 30
+   * days.
+   */
+  expireTime?: BackupTimestamp;
+}
+
+export interface CreateBackupOptions {
+  gaxOptions?: CallOptions;
+}
```

Comment:
Perhaps we should package all but `id` params in one object
```ts
export interface CreateBackupOptions {
  tableId: string; // tableId might be enough since we can build full table path
  expireTime: BackupTimestamp;
  gaxOptions?: CallOptions;
}
```

Similar to `creteInstance`, `createCluster`, `createTable` functions.